### DISCUSSION TITLE
Rapidly gaining on damage effects genes by pulling out gene splicer

### DISCUSSION THREAD

**Discussion by Xeavas** [FakeFishGames/Barotrauma#15270] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

You can rapidly gain vigor/hyperactivity by inducing multiple instances of organ damage if you have multiple gene effects. 
ie pulling out an advanced gene splicer with 4 gene effects with skitter genes give 240 units of hyperactivity

### Reproduction steps

1. Have 2 Compound gene with at least one containing mollosc/skitter
2. Put them all in an advanced gene splicer
3. Equip into gene slot and rip gene splicer out

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.7.0.1 (Unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Jasontti** [comment]
Tested a fix for this.
Couldn't trigger a buff from a gene being removed in any combination of combo or solo genes.
Closing discussion as resolved.
